lib/cisco_node_utils/evpn_multisite.rb

```diff
@@ -34,17 +34,31 @@ def initialize(multisiteid, instantiate=true)
     end
 
     def create
-      Feature.nv_overlay_enable
+      # There is an issue with EX platform using
+      # Feature.nv_overlay_enable generating error code '400'
+      # Using config_set for enabling nv overlay
+      config_set('feature', 'nv_overlay', state: '')
+      sleep 1
```

Comment:
Why do we need the sleep?

Reply:
I just used the logic for self.nv_overlay_enable in feature.rb. I think this is to allow the device to actually turn the feature on after processing the configuration and avoid timing issues.